README.md

```diff
@@ -80,5 +80,19 @@ Or you can execute the **build_macos.sh** to build the binaries.
 ### Build on Ubuntu 20.04
 You can execute the **build_linux.sh** to build the binaries.
 
-### Documentation
+## Documentation
 For documentation, please refer to the Wiki section.
+
+## Extra
+
+### Build compile db
```

Comment:
Use a more meaningful description: `### Generate Compilation Database`

Reply:
I see. It will be done in the next commit.